src/main/java/com/nickdsantos/onedrive4j/DriveService.java

```diff
@@ -175,8 +177,7 @@ public void downloadItemContent(String accessToken, DriveItem driveItem, OutputS
 
         try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
             HttpGet httpGet = new HttpGet(uri);
-            try (InputStream inputStream = httpClient.execute(httpGet).getEntity().getContent())
-            {
+            try (InputStream inputStream = httpClient.execute(httpGet).getEntity().getContent()) {
```

Comment:
I believe that this `execute()` call returns another object (`CloseableHttpResponse`?) which is supposed to be closed, but it isn't clear to me whether that is closed automatically by closing the client itself.